src/pages/candidate-details/candidate-details.ts

```diff
@@ -104,6 +105,7 @@ export class CandidateDetailsPage extends BasePageComponent implements OnInit, O
       this.store$.dispatch(new CandidateDetailsSlotChangeViewed(this.slotId));
     }
     this.store$.dispatch(new ClearChangedSlot(this.slotId));
+    this.candidateDetailsCheck.candidateDetailsViewed(this.slotId);
```

Comment:
We should be using the store to manage this state rather than a provider

Reply:
Worth getting rid of the provider entirely right?
Just using the store to remember when a candidate detail was viewed

Reply:
Yes would be a bit of re-work but probably worth it